ts/store/reducers/authentication.ts

```diff
@@ -17,31 +17,47 @@ import { GlobalState } from "./types";
 
 // Types
 
+type AuthenticationKinds =
+  | "LoggedOutWithoutIdp"
+  | "LoggedOutWithIdp"
+  | "LoggedInWithoutSessionInfo"
+  | "LoggedInWithSessionInfo";
+
+interface AuthenticationStateBaseInterface {
```

Comment:
why you need this?

Reply:
I create the base interface in order to type the `isSessionExpired` property. The `kind` property is present whatever the state, so it make sense to me to type it within the base interface too.